### PR TITLE
taskPopup

### PR DESCRIPTION
**Pull request by Olha3001**
[Demo](https://olha3001.github.io/Popup-1/) |
[Code base](https://github.com/Olha3001/Popup-1)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Olha3001** [comment]
Corrected

**Comment by Olha3001** [comment]
verified

File: submissions/Olha3001/Poppy/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <!--  пусть это будет лого) -->
+      <img src="./img-check/favicon.png" alt="" class="icon-nav-link check-icon_s logo" tabindex="1" >
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:13]: The logo should be a link and leads users to the home page.

File: submissions/Olha3001/Poppy/index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:8]: Add propper page title

File: submissions/Olha3001/Poppy/index.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+<body>
+    <header class="header">
+        <!--  пусть это будет лого) -->
+      <img src="./img-check/favicon.png" alt="" class="icon-nav-link check-icon_s logo" tabindex="1" >
+     <!-- навигация хедера -->
+      <nav>
+              <ul class="nav-menu">
+        <li class="list-nav"><a href="#" class="link-nav">Mail</a> </li>
+        <li class="list-nav"><a href="#" class="link-nav" >Images</a></li>
+        <li class="popup-menu">
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:19]: Fix code indentation, use [Prettier](https://prettier.io/) for it.

File: submissions/Olha3001/Poppy/index.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            <input type="checkbox" class="input-check" id="checkbox-menu">
+            <label for="checkbox-menu" class="label-container"><img src="./img-check/popup-button.png" class="check-icon_s icon-nav-link" tabindex="1">
+            </label>
+            <!-- контейнер для иконок-ссылок     -->
+                    <ul class="nav-item">
+        <li > 
+            <a href="#" class="check-item link" tabindex="0" ><img src="./img-check/calendar.png" alt="calendar" class="check-icons " >
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:26]: `tabindex="0"` on interactive element makes no sense.

File: submissions/Olha3001/Poppy/index.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            <p>Calendar</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/blogs.png" alt="blogs" class="check-icons" >
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:29]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/Olha3001/Poppy/index.html [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+            <a href="#" ><img src="./img-check/bell.png" alt="bell" class="check-icon_s icon-nav-link bell">
+            </a></li>
+       </div> 
+    </li>        
+</ul>
+    </nav>
+</header>
+   
+</body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:102]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/Olha3001/Poppy/index.html [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <!--  пусть это будет лого) -->
+      <img src="./img-check/favicon.png" alt="" class="icon-nav-link check-icon_s logo" tabindex="1" >
+     <!-- навигация хедера -->
+      <nav>
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:15]: Add mobile version
image

File: submissions/Olha3001/Poppy/index.html [hunk 8]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```
Comment (A-Ostrovnyy) [inline submissions/Olha3001/Poppy/index.html:1]: By default your popup is open, it must be closed, also, fix the order of interactive elements.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Pull request closed without merging**